compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -155,7 +155,19 @@ void set_minmax_to_non_const(loco::Graph *g, float min, float max)
   }
 }
 
-class LogisticTestGraph final : public luci::test::TestIOGraph
+/**
+ * @brief Simple Test Graph
+ * @note
+ * The simple test graph's nodes are initialized with
+ * simplely-choosen shapes and values.
```

Comment:
```suggestion
 * @note
 * The simple test graph's nodes are initialized with
 * simple shapes and values.
```

Or, I think this note can be removed at all.

Reply:
@jinevening I wanted to include the meaning that those `SimpleTestGraph` is just one instance, not like TestIOGraph.
I was worried that the name doesn't imply enough, so I put that note.

Reply:
So I may leave a note with the change :-D